run.sh

```diff
@@ -48,7 +48,7 @@ DOCKER_RUN_COMMAND="ansible-galaxy install -r /tmp/src/requirements.yml --roles-
 docker run --rm -i \
     -v $(pwd):/tmp/src:z \
     -v $HOME/.kube:/root/.kube:z \
-    -t redhatcop/openshift-applier:v3.9.0 \
+    -t redhatcop/openshift-applier:v2.0.5 \
```

Comment:
since this image has moved to [quay.io](https://quay.io/repository/redhat-cop/openshift-applier?tab=tags), it changed name to `redhat-cop`, so please add the `-` in the name. + to make sure it works on all target machines, please add `quay.io/` in front of it... i.e.: `quay.io/redhat-cop/openshift-applier:v2.0.5`

Reply:
Thanks for catching that. Fixed!